src/rules/form-control-has-label.ts

```diff
@@ -1,31 +1,38 @@
 import type { Rule } from "eslint";
 import type { AST } from "vue-eslint-parser";
 
+interface FormControlHasLabelOptions {
+  labelComponents: string[];
+}
+
 import {
   defineTemplateBodyVisitor,
   getElementAttributeValue,
   getElementType,
   hasAriaLabel,
   isAriaHidden,
-  makeDocsURL
+  makeDocsURL,
+  makeKebabCase,
 } from "../utils";
 
 function isLabelElement(
   node:
     | AST.VElement
     | AST.VDocumentFragment
     | AST.VText
-    | AST.VExpressionContainer
+    | AST.VExpressionContainer,
+  options: FormControlHasLabelOptions
```

Comment:
(optional)
```suggestion
  { labelComponents = [] }: FormControlHasLabelOptions
```

then the line 26 can be like:
```js
  const labelComponents = labelComponents.map(makeKebabCase).concat("label");
```

Reply:
Tweaked slightly to prevent overwriting the variable